### ISSUE TITLE
virtualizationThreshold not working? and a bug when bigger then data list

### ISSUE THREAD

**Issue by Ruud-cb** [angular-ui/ui-grid#5588]
Noticed when `virtualizationThreshold` is set to a larger or equal number then the data length, it will cause serious unresponsive lag. The plunker, which is a fork of issue #5585 (scroll lag), will have a hard time  loading. The `virtualizationThreshold` is set to 1001, data list begins with 1000 and increases. When the data list is larger then 1001 it returns back to normal, with scroll lag.

https://embed.plnkr.co/wmxIyxz7GGQc1r8oytUf/

That's one issue, the other issue is that it is not working, I guess? As far as I know the `virtualizationThreshold` should render the amount of rows it is set to? Yet it seems it does not render more than is visible because it still shows white rows when scrolling (thus rendering new rows). I might be wrong here?

**Comment by angular-git-user** [comment]
Yes the virtualization threshold will render number of rows it has been set to... making the grid unresponsive if there are too many rows bcoz it has to create dom for each row... the best approach is to leave the virtualizaiton threshold to default.. and let the grid render each row has you scroll down.. since this was the primary feature of ui-grid..

**Comment by orneryd** [comment]
Thanks for contributing! This repository has been transferred, and we're preparing the new 5.x release. We're closing older issues and pull requests as part of the reset. Please join us on Discord to follow the work or continue the conversation: https://discord.gg/Baz4w8ZWN
